# The MPI example includes aocommon headers that need the xtensor headers, but did not link against xtensor;
# this fails to compile when building with -DBUILD_WITH_MPI=ON
--- IDG.orig/idg-bin/examples/cxx/distributed/CPU/CMakeLists.txt
+++ IDG/idg-bin/examples/cxx/distributed/CPU/CMakeLists.txt
@@ -16,7 +16,7 @@
   PRIVATE ${FTI_INCLUDE_DIR})
 
 # Set link libaries
-target_link_libraries(${PROJECT_NAME} PRIVATE idg-util idg-cpu)
+target_link_libraries(${PROJECT_NAME} PRIVATE idg-util idg-cpu xtensor)
 target_link_libraries(${PROJECT_NAME} PRIVATE ${MPI_mpi_LIBRARY})
 target_link_libraries(${PROJECT_NAME} PRIVATE ${MPI_mpi_cxx_LIBRARY})
 if(${BUILD_WITH_FTI})
